video/video_receive_stream.cc

```diff
@@ -82,6 +82,12 @@ VideoCodec CreateDecoderVideoCodec(const VideoReceiveStream::Decoder& decoder) {
 
 namespace internal {
 
+const char kLogLatencyToFileFieldTrial[] = "OWT-Log-Latency-To-File";
```

Comment:
Can we share constant string with modules/pacing/paced_sender.cc?

Reply:
done.